online-devops-dojo/devops-kaizen/assets/github-issues.yaml

```diff
@@ -0,0 +1,30 @@
+---
+title: Implement Continuous Testing
+body: >
+  From **Tina**:
+  ![](https://s3.amazonaws.com/devopsdojoassets/tina.png)
+
+  # DevOps Kaizen Improvement Theme
+
+  ### **:green_book:  Now / Current State**
+
+  Testing is most manually, mainly performed after the development is completed, feedback on issues is too late in the process which in turn is leading to rework and delays
+
+  ### **:star: Definition of awesome**
+
+  Any code committed to our source code management system would be continuously integrated via a Jenkins pipeline. 
+
+  The pipeline would execute of a comprehensive set of automated tests (continuous testing) against the code, these tests would include unit tests that cover 70 to 80% of the code and automated acceptance tests of key functional areas.
+
+  ### **:dart: Next target condition**
+
+  Introduce Unit Testing and execute the unit tests from a CI pipeline.
+
+  ### **:feet: First steps**
+
+  Create the Pipeline, have it invoked when code is committed and execute some unit tests from the pipeline
```

Comment:
Maybe a first step recommended to Create unit tests? i've found in my coaching sometimes we jump to the audomation (the pipeline) to improve but we have nothing to actually test yet, and we want the students to wlak before they run. also helps break down into smaller batch sizes to be more practical for an actual backlog.  given that, suggest a bullet to "Create a set of unit tests that can be executed offline" maybe? before the next step of putting them into a CI/CD pipeline?

Reply:
@adamdmharvey no issues with that feel free to submit a PR to that effect